src/server/actions/study-job.actions.ts

```diff
@@ -65,7 +71,7 @@ export const rejectStudyJobResultsAction = memberAction(async (info) => {
     revalidatePath(`/member/[memberIdentifier]/study/${info.studyId}/review`)
 }, minimalJobInfoSchema)
 
-export const dataForJobAction = memberAction(async (studyJobIdentifier) => {
+export const dataForJobAction = userAction(async (studyJobIdentifier) => {
```

Comment:
While it's always hard to come up with good names, a good rule is that methods should never be called `data` 😁 instead it's better to use a more descriptive name.  Maybe this could be `loadStudyJobAction` ?